css/main.css

```diff
@@ -0,0 +1,100 @@
+:root {
+    --left-pad: 7%;
+}
+html,
+body {
+    font-family: "fira-sans", Helvetica, Arial, sans-serif;
+    height: 100%;
+}
+div, 
```

Comment:
Try to use classes rather than tags to apply styles in case of browser performance reasons and compatibility with other styles